config/initializers/feature_toggle.rb

```diff
@@ -0,0 +1,9 @@
+class FeatureToggle
+  def self.needs_question_enabled?
+    !Rails.env.production?
+  end
+
+  def self.show_schemes_button?
```

Comment:
I think this should probably always follow a set pattern of: `true` if you want the feature on, `false` if you don't.

This seems like for `needs_question_enabled?` true is on but for `show_schemes_button` true is actually off?

Reply:
This is probably down to poor naming on my part - I wanted the needs question to be enabled on non prod environments

Reply:
Will amend this to follow the suggested pattern 😄

Reply:
@baarkerlounger I've amended the code to fix this - it was structured in a way that returning true here would hide the button so I've update the condition to display the button

Reply:
@baarkerlounger commit is here: https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/661/commits/120b7dde9aa035e07fa959ca1b7bf69d49de3142 

check the changes in `app/components/primary_navigation_component.rb`, this amendment should now make it line up with your expectations